modules/creating-a-service-account-in-your-project.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/assuming-an-iam-role-using-the-pod-identity-webhook.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-service-account-in-your-project_{context}"]
+= Creating a service account in your project
+
+Add a service account in your user-defined project. Include an `eks.amazonaws.com/role-arn` annotation in the service account configuration that references the Amazon Resource Name (ARN) for the AWS Identity and Access Management (IAM) role that you want the service account to assume.
+
+.Prerequisites
+
+* You have created an AWS IAM role for your service account. For more information, see _Setting up an AWS IAM role for a service account_.
+* You have access to a {product-title} with AWS Security Token Service (STS) cluster. Admin-level user privileges are not required.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. In your {product-title} cluster, create a project:
++
+[source,terminal]
+----
+$ oc new-project <project_name> <1>
+----
+<1> Replace `<project_name>` with the name of your project. The name must match the project name that you specified in your AWS IAM role configuration.
++
+[NOTE]
+====
+You are automatically switched to the project when it is created.
+====
+
+. Add the following service account configuration to a file named `test-service-account.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <service_account_name> <1>
+  namespace: <project_name> <2>
+  annotations:
+    eks.amazonaws.com/role-arn: "<aws_iam_role_arn>" <3>
```

Comment:
The OIDC of OCP STS cluster looks like this
arn:aws:iam::301721915996:oidc-provider/jshu-1116sts-2911-oidc.s3.us-east-2.amazonaws.com
vs. EKS cluster one
"arn:aws:iam::111122223333:oidc-provider/oidc.REGION.eks.amazonaws.com/CLUSTER_ID"

So for OCP STS cluster, still set annotation to "eks.amazonaws.com/role-arn: "???

Reply:
My understanding is that the pod identity webhook is looking for the `eks.amazonaws.com/role-arn` annotation. 

@sjenning can you please advise? Thanks!

Reply:
https://github.com/openshift/cloud-credential-operator/blame/c34501f3d66f132d21a2a9620a886c4b144e1571/bindata/v4.1.0/aws-pod-identity-webhook/deployment.yaml#L32

Yes, unfortunately we never changed it before we rolled it out GA so now it is stuck that way

For built in OCP components, we do not use the webhook to inject role information since we can't be sure the CCO is running by the time the role is used.

Reply:
Thank you for confirming that. I will subsequently keep the annotation as it is in the code block and will set this comment thread to resolved. Thanks again!